src/environments/environment.ts

```diff
@@ -5,8 +5,8 @@
 
 export const environment = {
     production: false,
-    ccda_validation_url: 'https://james.test.sitenv.org/referenceccdaservice/', // hhs test james.test.sitenv.org
-    ccda_r2_cures_validation_url: 'https://ccda.test.sitenv.org/referenceccdaservice/', // hhs dev ccda.test.sitenv.org 34.236.48.201 ONCSITE-TEST-CCDA
+    ccda_validation_url: 'https://james.healthit.gov/referenceccdaservice/',
```

Comment:
This should probably be james.dev.sitenv.org. But, it's fine for now. We can always change if needed to test dev vs prod. We may even be shutting this validator down...